pkg/client/userd/grpc.go

```diff
@@ -56,20 +56,20 @@ func withReqNumber(ctx context.Context, num int64) context.Context {
 	return context.WithValue(ctx, reqNumberKey{}, num)
 }
 
-func (s *service) callCtx(ctx context.Context, name string) context.Context {
+func (s *Service) callCtx(ctx context.Context, name string) context.Context {
```

Comment:
Why does the grpc service need to be exported? The  methods here should normally be called from grpc.

Reply:
I believe is because we are taking the struct from `userd/service.go`